src/etc/inc/services.inc

```diff
@@ -1153,9 +1155,10 @@ EOD;
 
 				if (isset($sm['ddnsupdate'])) {
 					if (($sm['ddnsdomain'] <> "") && ($sm['ddnsdomain'] != $dhcpifconf['ddnsdomain'])) {
-						$smdnscfg .= "		ddns-domainname \"{$sm['ddnsdomain']}\";\n";
+						$smdnscfg .= "        ddns-domainname \"{$sm['ddnsdomain']}\";\n";
```

Comment:
Why change from tabs to spaces? The rest of the file uses tabs.

Reply:
I changed it because the resulting output was indented differently than the rest. With that change all configuration options are in line.

Reply:
This looks like it would result in a mix of spaces and tabs, which is not desirable. If something else is using spaces, it should be changed to tabs and not the other way around.